### PR TITLE
Example updates.

### PR DESCRIPTION
**Pull request by Shlol762**
Just connecting the example to nextcord.

### CODE REVIEW

File: examples/secure_token_storage.py [hunk 1]
```diff
@@ -2,13 +2,15 @@
 # You should load the token as an environment variable, then access it like this:
 
 import os
+from nextcord.ext.commands import Bot
```
Comment (TAG-Epic) [inline examples/secure_token_storage.py:5]:
```suggestion
from nextcord.ext import commands
```

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

**Review by TAG-Epic — APPROVED** [review]

File: examples/secure_token_storage.py [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
 import os
+from nextcord.ext import commands
 
 # There are other ways to load environment variables but this is one
 from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
 
 # load_dotenv reads from a file called .env in the same directory as the python files which should roughly look like TOKEN="1234567890"
 load_dotenv()
 token = os.getenv('BOT_TOKEN')
+bot = commands.Bot()
```
Comment (DenverCoder1) [inline examples/secure_token_storage.py:13]: `commands.Bot()` does not have a default value for `command_prefix` so I think we should not be leaving that out in the example, unless something like #101 gets merged.

Comment (ooliver1) [reply examples/secure_token_storage.py:13]: That won't be merged as vco had a pretty good point, maybe making the prefix something simple like ! Or ?

File: examples/secure_token_storage.py [hunk 3]
```diff
@@ -2,13 +2,15 @@
 # You should load the token as an environment variable, then access it like this:
 
 import os
+from nextcord.ext import commands
 
 # There are other ways to load environment variables but this is one
 from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
 
 # load_dotenv reads from a file called .env in the same directory as the python files which should roughly look like TOKEN="1234567890"
```
Comment (ooliver1) [inline examples/secure_token_storage.py:10]: Oh yeah one thing, that comment is incorrect "should look like TOKEN=" but the env taken is BOT_TOKEN

File: examples/secure_token_storage.py (hunk 2 above)
Comment (TAG-Epic) [reply examples/secure_token_storage.py:13]: Yeah I agree we need a example prefix here. Maybe use the same as in the other examples?

Comment (H4ckerxx44) [reply examples/secure_token_storage.py:13]: I would suggest "?" as it is the prefix of all other examples

Comment (TAG-Epic) [reply examples/secure_token_storage.py:13]: > I would suggest "?" as it is the prefix of all other examples

We have been using "$" everywhere else.

Comment (H4ckerxx44) [reply examples/secure_token_storage.py:13]: Oh whoops, I meant $, lol.

**Pull request merged**